examples/self/run.bat

```diff
@@ -2,6 +2,6 @@
 set curr_dir=%cd%
 chdir /D "%~dp0"
 
-node ..\..\bin\typedoc --includeDeclarations --externalPattern **/lib/** --mode file --name "TypeDoc Documentation" --out doc\ ..\..\src\
+node ..\..\bin\typedoc --includeDeclarations --externalPattern **/lib/** --mode file --name "TypeDoc Documentation" --out doc\ ..\..\src\lib\
```

Comment:
Should this be changed to invoke TypeDoc with the same arguments as the unix side?